src/components/project/enable-encryption.vue

```diff
@@ -298,7 +300,8 @@ export default {
             [
               "Encryption cannot be turned off once enabled.",
               "In a future version, you will be able to disable encryption, which will decrypt your data. This will be true even if you enable encryption now."
-            ]
+            ],
+            "Draft Submissions from all Forms will be deleted."
```

Comment:
Small ideas for the text:

- For someone who doesn't remember what "Draft Submissions" are, can we connect them to draft testing? I see 1 use of the term "Draft Submission", 1 use of the term "Draft test Submission", and 5 uses of the term "test Submission". Here, how about we use either "test Submissions" or "Draft test Submissions"?
- Other places mostly talk about "removing" draft submissions rather than "deleting" them. How about we match that language?
- Could we add "permanently"?

Altogether, how about something like:

> Test Submissions to existing Draft Forms will be permanently removed.